### ISSUE TITLE
Setting featureStyle removes ability to edit a polygon

### ISSUE THREAD

**Issue by yurivangeffen** [uber/nebula.gl#369]
#### Describe the bug
When I customize the style of a shape through featureStyle, I no longer have the ability to create new vertices on the polygon.
#### Actual Result
Nothing happens when clicking a line between vertices.
#### Expected Result
To be able to create new shapes by adding vertices, also when using a custom style.
#### Screenshots
![Apr-02-2020 14-52-17](https://user-images.githubusercontent.com/4107720/78251505-b886be00-74f1-11ea-8bea-d909cf9bd954.gif)
Correct behaviour.

![Apr-02-2020 14-52-42](https://user-images.githubusercontent.com/4107720/78251493-b15fb000-74f1-11ea-9647-1290eddb03e9.gif)
Incorrect behaviour.

#### To Do List 
  - [ ] Add label and assign to milestone
  - [ ] Coding
  - [ ] Test

**Comment by supersonicclay** [comment]
Are you using `react-map-gl-draw`? Could you create a repro on codepen or codesandbox.io?

**Comment by yurivangeffen** [comment]
Yes, it's react-map-gl-draw. I'll whip up a minimal example.

**Comment by xintongxia** [comment]
@yurivangeffen did you provide a `clickRadius` prop to `react-map-gl-draw` editor?

**Comment by yurivangeffen** [comment]
Yes, I did. But removing it doesn't change anything.

**Comment by yurivangeffen** [comment]
Here's an example: https://codesandbox.io/s/react-map-gl-draw-example-tzhy4
The only thing I changed to your example is adding 
```javascript
featureStyle={{
   fill: "blue"
}}
```
to the Editor. If you now draw a polygon, change to edit mode, and try to add another node somewhere in the polygon (like in the gif above) you immediately go into drag mode. I would expect none of the core edit functionality to change when editing a feature style.

**Comment by xintongxia** [comment]
I'll have a PR to fix this. At the meantime, a workaround is provide a `stroke` prop to `featureStyle`

```
featureStyle={{
  fill: "blue",
  stroke: 'rgb(33, 33, 33)'
}}
```

**Comment by xintongxia** [comment]
Should be fixed in v0.18.4
